src/kubernetes_backup/kubernetes_backup/s3.py

```diff
@@ -4,7 +4,7 @@
 
 import boto3
 
-class S3Interface(object):
+class S3Interface():
```

Comment:
You don't need the brackets here... I'm not sure if the parser cares.

Reply:
oh wow. this mistake has been proliferating. thanks for pointing out. fixed.